bitmap-mosnter/test/bitmaptest.js

```diff
@@ -0,0 +1,115 @@
+'use strict';
+const readwrite = require(__dirname + '/../lib/readbitmap');
+const bitobject = require(__dirname + '/../lib/parser');
+const transform = require(__dirname + '/../lib/transform');
+const expect = require('chai').expect;
+
+describe('testing the reading and writing functions', function() {
+  describe('testing output of readbitmap.js and newbitmapfile.js', function() {
+    it('should return a buffer of hex', function(done) {
+      readwrite.bitmapReader(__dirname  + '/../bitmap1.bmp', function(err, data) {
+        expect(err).to.equal(null);
+        expect(data.toString('hex', 0, 14)).to.equal('424d462b00000000000036040000');
+        done();
+      });
+    });
+  });
+});
```

Comment:
I don't see tests for newbitmapfile here. Don't forget to test all outputs of your modules.